libs/sdk-backend-base/src/normalizingBackend/normalizer.ts

```diff
@@ -403,6 +404,15 @@ export class Normalizer {
                     ref.localIdentifier = this.normalizedLocalId(ref.localIdentifier);
                 }
             }
+
+            if (isRankingFilter(filter)) {
+                const measureRef = filter.rankingFilter.measure;
+                const attributeRefs = filter.rankingFilter.attributes || [];
```

Comment:
small stuff, you can rewrite this using object destructuring:

```
const { measure, attributes = [] } = filter.rankingFilter;
```

Reply:
fixed